Signal/src/ViewControllers/ConversationView/ConversationLayoutInfo.swift

```diff
@@ -40,6 +63,8 @@ public class ConversationLayoutInfo: NSObject {
     @objc public var maxMessageWidth: CGFloat = 0
     @objc public var maxFooterWidth: CGFloat = 0
 
+    @objc public var textInsets = OWSDirectionalEdgeInsets()
```

Comment:
nit: I think a `.zero` class initializer like other edge inset classes would be preferable to default values, no biggie though.

Reply:
Done.